spyder-custom-layout/requirements/conda.txt

```diff
@@ -0,0 +1,4 @@
+qtawesome
```

Comment:
```suggestion
spyder >=5.0.4
qtawesome
```

Please don't add this restriction in our cookiecutter, just the dependency in Spyder.